### PR TITLE
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,31 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0a902c2fbf9e455f88c61bd0e75fed09)](https://app.codacy.com/gh/Seagate/cortx-fs-ganesha?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-fs-ganesha&utm_campaign=Badge_Grade_Settings)
 
-# CORTXFS-FSAL
+# CORTX-FS-Ganesha
 
-NFS Ganesha backend for CORTXFS implementation.
+The CORTX-FS-GANESHA submodule provides the FSAL code for CORTX-FS that is used by the CORTX-NFS-Ganesha repository. In other words, CORTX-FS-Ganesha is the backend for CORTX-FS implementation. The FSAL (Filesystem Abstraction Layer) implements the NFS-Ganesha FSAL-API based on CORTX-FS as the underlying filesystem.
```
Comment (sachinpunadikar) [inline README.md:5]: submodule -> repo
CORTX-FS  -> CORTXFS

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please work on the suggestions

File: README.md (hunk 1 above)
Comment (Saumya-Sunder) [reply README.md]: Done @sachinpunadikar, please check.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,31 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0a902c2fbf9e455f88c61bd0e75fed09)](https://app.codacy.com/gh/Seagate/cortx-fs-ganesha?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-fs-ganesha&utm_campaign=Badge_Grade_Settings)
 
-# CORTXFS-FSAL
+# CORTXFS-Ganesha
 
-NFS Ganesha backend for CORTXFS implementation.
+The CORTXFS-GANESHA repository provides the FSAL code for CORTXFS that is used by the CORTX-NFS-Ganesha repository. In other words, CORTXFS-Ganesha is the backend for CORTXFS implementation. The FSAL (Filesystem Abstraction Layer) implements the NFS-Ganesha FSAL-API based on CORTXFS as the underlying filesystem.
```
Comment (sachinpunadikar) [inline README.md:5]: Better wording would be:
The CORTXFS-GANESHA repository provides CORTXFS FSAL (Filesystem Abstraction Layer) code. This code implements NFS Ganesha FSAL-APIs, based on CORTXFS as the underlying filesystem. NFS Ganesha makes calls into CORTXFS FSAL code and CORTXFS FSAL code interacts with CORTXFS.

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please do the suggested changes.

**Comment by sachinpunadikar** [comment]
Also look at the codacy findings and fix them .. https://app.codacy.com/gh/Seagate/cortx-fs-ganesha/pullRequest?prid=6285972

File: README.md (hunk 2 above)
Comment (Saumya-Sunder) [reply README.md]: Done. Codacy fails for all documents by default. you'll need to ignore the file for Codacy checks @sachinpunadikar

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by s-arya — APPROVED** [review]

**Pull request merged**